modules/engines/engine-hdb/src/test/resources/HdbtableHanaXSAdvancedContent.hdbtable

```diff
@@ -0,0 +1,5 @@
+COLUMN TABLE "::HdbtableHanaXSAdvancedContent" (
```

Comment:
Is the table named like this on purpose?

Reply:
The table name should match the name of .hdbtable file.